documentation/modules/ROOT/pages/connectors/oracle.adoc

```diff
@@ -4210,6 +4209,275 @@ After the connector's commit SCN advances beyond the DDL change, to prevent unpa
+
+[[oracle-openlogreplicator-configuration]]
+=== OpenLogReplicator configuration
+
+OpenLogReplicator is a third-party standalone process that is responsible for reading the Oracle redo and archive logs.
+Unlike other {prodname} Oracle adapters, this means that there are two configurable components that must be configured independently of one another.
+
+OpenLogReplicator is configured by using a JSON-based configuration in a file called `scripts/OpenLogReplicator.json`.
+The format of this JSON configuration can be found https://github.com/bersler/OpenLogReplicator/blob/master/documentation/reference-manual/reference-manual.adoc#openlogreplicator-json-file-format[here].
+
+.OpenLogReplicator configuration
+[source,json,indent=0]
+----
+{
+  "version": "1.2.1",
+  "source": [{
+    "alias": "SOURCE",
+    "name": "ORACLE", <.>
+    "reader": {
+      "type": "online",
+      "path-mapping": ["/opt/olr/recovery_area", "/opt/olr/recovery_area/ORCLCDB/archivelog"], <.>
+      "user": "c##dbzuser", <.>
+      "password": "dbz", <.>
+      "server": "//<ip>:<port>/ORCLPDB1" <.>
+    },
+    "format": { <.>
+      "type": "json",
+      "column": 2,
+      "db": 3,
+      "interval-dts": 9,
+      "interval-ytm": 4,
+      "message": 2,
+      "rid": 1,
+      "schema": 7,
+      "scn-all": 1,
+      "timestamp-all": 1
+    }
+  }],
+  "target": [{
+    "alias": "DEBEZIUM",
+    "source": "SOURCE",
+    "writer": {
+      "type": "network", <.>
+      "uri": "<host>:<port>" <.>
+    }
+  }]
+}
+----
+<.> This should match the `openlogreplicator.source` connector configuration.
+<.> List of file path pairs `[before1,after1,befor2,after2,...]` where if a log file path matches one of the `beforeX` prefixes, the prefix is replaced with the `afterX` path.  This is useful when OpenLogReplicator is running on a different host than the source database and the path to the redo and archive logs different between the database and the OpenLogReplicator process.
+<.> This should match the `database.user` connector property.
+<.> This should match the `database.password` connector property.
+<.> This should point to the database host, port, and Oracle SID.
+<.> This specifies the payload format ingested by {prodname} Oracle connector. Use these values as specified, as these are the only format options that we require beyond the defaults.
+<.> This must specify `network`, as {prodname} Oracle connector communicates with OpenLogReplicator via a network connection.
+<.> This specifies the bind host and port that OpenLogReplicator listens for connections on.  This should be accessible by {prodname} Oracle connector.
+
+[[oracle-openlogreplicator-connector-properties]]
+=== OpenLogReplicator connector properties
+
+The following configuration properties are _required_ when using OpenLogReplicator unless a default value is available.
```

Comment:
```suggestion
The following configuration properties are required when using OpenLogReplicator, unless a default value is available.
```

The table indicates that none of these properties have default values. Can we remove the last part of the preceding sentence? Or should we keep it in case we later implement defaults?

Reply:
I think we can remove it for now; however, I want to point out that we use the `_required_` italic text elsewhere in the documentation.  Should we also adjust those other lines?

Reply:
Technically, I don't think that it's necessary, but as a rule, I like to err on the side of minimizing the use of text formatting to emphasize content.

Reply:
To be clear, what I meant was in all our other connector properties sections in the Oracle documentation, we use the same phrasing with the same formatting.  Should we align those to be uniform?

Reply:
Ok, got it. No, let's just leave everything as-is for now. It really ain't broke, so let's not fix it.